ocs_ci/utility/kms.py

```diff
@@ -599,15 +599,19 @@ def vault_create_policy(self, policy_name=None):
         # Printing out here the Vault Path Token
         logger.info(f"Vault Path Token is: ${self.vault_path_token}")
 
-    def generate_vault_token(self):
+    def generate_vault_token(self, ttl="768h"):
         """
         Generate a token for self.vault_policy_name
 
         Returns:
             str: vault token
 
         """
-        cmd = f"vault token create -policy={self.vault_policy_name} " f"--format=json"
+        cmd = (
+            f"vault token create -policy={self.vault_policy_name} "
+            f"--format=json "
```

Comment:
There is a f-string without any variable, which I think I think this should be catch by pre-commit hook.